iotix/patterns/header.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Title: header
+ * Slug: iotix/header
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"bottom":"var:preset|spacing|70","right":"5vw","left":"5vw"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-right:5vw;padding-bottom:var(--wp--preset--spacing--70);padding-left:5vw"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"40px","right":"0","bottom":"40px","left":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide" style="padding-top:40px;padding-right:0;padding-bottom:40px;padding-left:0"><!-- wp:site-title /-->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:navigation {"layout":{"type":"flex","justifyContent":"left","orientation":"horizontal"}} /-->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"fontSize":"small"} -->
+<div class="wp-block-button has-custom-font-size has-small-font-size"><a class="wp-block-button__link wp-element-button"><?php esc_html_e( 'Sign In', 'iotix' ); ?></a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
```

Comment:
Do we need the 3 nested group blocks here in the header pattern?

Reply:
I think these are expected, yes. Top is used to set innerblock content width. 2nd is a row block to make items flow horizontally. 3rd is the button and nav items on the right:

![Screenshot 2024-07-16 at 17 15 14](https://github.com/user-attachments/assets/6cda9dd4-c493-4b78-b9f6-edd94da5dc0c)